### ISSUE TITLE
Rule: audio-caption changes

### ISSUE THREAD

**Issue by jkodu** [dequelabs/axe-core#1142]
The following test cases should report as `inapplicable` but is shown as `incomplete` for the rule `audio-caption`.

Audio without controls:
```html
<audio src="../test-assets/moon-audio/moon-speech.mp3"></audio>
```

Audio with hidden controls:
```html
<audio src="../test-assets/moon-audio/moon-speech.mp3" controls style="display: none;"></audio>
```

Note: these tests should be reported as `semi-automated`

**Comment by jkodu** [comment]
Closing, no change required.
